cli/src/cli.rs

```diff
@@ -68,11 +72,14 @@ pub enum Commands {
         #[clap(short, long, action)]
         last_finalization_proof: String,
     },
-    /// Print the information about the block data Git server that this node is hosting.
+    /// Print the information about the Git server that this node is hosting.
     Git,
     /// Clean the repository, removing all the outdated (incompatible with `finalized`) commits.
     Clean {
-        /// If enabled, it will remove all the commits except those included in `finalized`.
+        /// If enabled, it will remove
+        /// 1. all branches except `finalized`, `fp`, and `work`
+        /// 2. all remote repositories
+        /// 3. all orphan commits
```

Comment:
```suggestion
        /// 1. all branches except `finalized`, `fp`, and `work`
        /// 2. all orphan commits
```
I think this is better since remote repositories only exist as remote branches. If you want to emphasize the deletion of remotes, how about adding a detail to 1?

Reply:
In what way specifically? Nothing comes to my mind now.

Reply:
Run `git remote --verbose`. That's the list of 'remote repositories'. Not all remote branches from the remote repository automatically exist in the local repository unless fetching them.

Reply:
Discussed in Discord and well understood.